src/app/dim-ui/CharacterSelect.m.scss

```diff
@@ -42,9 +36,15 @@
 }
 
 .tile {
+  height: 46px;
   min-width: 230px;
   @include phone-portrait {
+    min-width: 260px;
```

Comment:
On mobile you could avoid the cropped background by also adjusting the height

Reply:
Ah yeah! I hope to redesign the entire sticky header it to take up less space, but for now maybe I'll make the tile narrower again.